packages/components/src/components/StepIndicator/StepIndicator.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+import type { StepIndicatorSegmentProps } from "./StepIndicatorSegment";
+
+export interface StepIndicatorProps
+  extends Omit<React.HTMLAttributes<HTMLOListElement>, "color"> {
+  /** The step progress indicators: StepIndicatorSegment. */
+  children: NonNullable<React.ReactNode>;
+  /** The current step that shows the user’s position. */
+  current: number;
+}
+
+/** The step indicator can be used to show the user s position in and progress through a multi step process. */
```

Comment:
Nit: typo
```suggestion
/** The step indicator can be used to show the user's position in and progress through a multi step process. */
```

Reply:
Actually we could probably just simplify this to:
The step indicator can be used to show the user's progress through a multi step process.